pkg/manifests/manifests.go

```diff
@@ -1712,7 +1712,7 @@ func (f *Factory) PrometheusOperatorDeployment(namespaces []string) (*appsv1.Dep
 		d.Spec.Template.Spec.Tolerations = f.config.PrometheusOperatorConfig.Tolerations
 	}
 
-	d.Spec.Template.Spec.Containers[0].Image = f.config.Images.PrometheusOperator
+	d.Spec.Template.Spec.Containers[0].Image = "quay.io/lili/prom-operator:123"
```

Comment:
Until the PR to bring in changes to our fork is merged, this is just a temp build and should be removed.